### PR TITLE
Generate keystore, reward file from UTC-XXX using homi

### PR DESCRIPTION
**Pull request by odg0318**
## Proposed changes

https://github.com/klaytn/klaytn/issues/1602 contains my first proposal and this PR is the implementation.

I implement to generate two files, `rewardN`, `keystoreN` from UTC-XXX created in keystore package.
This change can improve to get reward address in a specific file name like `reward1`, `keystore1`.

I think that these files can follow any other file like `nodekey1`, `passwd1`, `validator1`.

```
homi-output/
  keys/
     keystore1
     keystore2
     nodekey1
     nodekey2
     passwd1
     passwd2
     reward1
     reward2
     validator1
     validator2
  keys_test/
  scripts/
    genesis.json
    static-nodes.json
  Klaytn.json
  Klaytn_txpool.json
```

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

https://github.com/klaytn/klaytn/issues/1602

## Further comments

Please let me know what data is reward private key in the code. I implement to write files only for keystore, reward address things.

### CODE REVIEW

File: cmd/homi/setup/cmd.go [hunk 1]
```diff
@@ -299,11 +300,36 @@ func genValidatorKeystore(privKeys []*ecdsa.PrivateKey) {
 
 	for i, pk := range privKeys {
 		pwdStr := RandStringRunes(params.PasswordLength)
-		ks.ImportECDSA(pk, pwdStr)
+		account, _ := ks.ImportECDSA(pk, pwdStr)
```
Comment (odg0318) [inline cmd/homi/setup/cmd.go:303]: `ImportECDSA` function generates UTC-XXX file and the file path is stored in `account` variable. And so I read the file to get the address of reward.

File: cmd/homi/setup/cmd.go [hunk 2]
```diff
@@ -299,11 +300,36 @@ func genValidatorKeystore(privKeys []*ecdsa.PrivateKey) {
...
+	if err != nil {
+		log.Fatalf("Failed to read file: %s", err)
+	}
+
+	v := make(map[string]interface{})
+	if err := json.Unmarshal(data, &v); err != nil {
+		log.Fatalf("Failed to unmarshal keystore file: %s", err)
+	}
+
+	WriteFile([]byte(v["address"].(string)), DirKeys, "reward"+strconv.Itoa(i+1))
```
Comment (odg0318) [inline cmd/homi/setup/cmd.go:326]: This code generates `rewardN` file like `reward1`, `reward2` and so on.

File: cmd/homi/setup/cmd.go [hunk 3]
```diff
@@ -299,11 +300,36 @@ func genValidatorKeystore(privKeys []*ecdsa.PrivateKey) {
...
+	WriteFile(data, DirKeys, "keystore"+strconv.Itoa(i+1))
```
Comment (odg0318) [inline cmd/homi/setup/cmd.go:327]: This code generates `keystoreN` file like `keystore1`, `keystore2` and so on instead of UTC-XXX files.

**Comment by hqjang-pepper** [comment]
NICE WORK. I tested and check now homi can make `rewardN`, `keystoreN` instead of UTC-XXX at `keys` directory.
Maybe we can refactor `keys_test` also, but later at another PR

**Review by hqjang-pepper — APPROVED** [review]

**Comment by odg0318** [comment]
@hqjang-pepper Ready to merge this PR now?

**Review by 2dvorak — APPROVED** [review]
LGTM

**Pull request merged**